src/app/dynamic/enterprise/default-constraints/template.html

```diff
@@ -0,0 +1,14 @@
+<!--
+Copyright 2020 The Kubermatic Kubernetes Platform contributors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<km-default-constraint-list [displayedColumns]="displayedColumns"></km-default-constraint-list>

```

Comment:
Since `DefaultConstraintComponent` is in shared module I think it makes no sense to use dynamic modules to specify different `displayedColumns`. I think in this case we can put this logic inside the `DefaultConstraintComponent` itself since there is not much to hide there.

Reply:
Sounds good to me. 👍 
So basically we then use the `getEditionDirName()` function directly in the `DefaultConstraintComponent` to decide which columns we want to display, right?

Reply:
Yep, exactly.